notebooks/PGx - CYP2C19 example 2.ipynb

```diff
@@ -0,0 +1,516 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "5040a57e-c81e-45da-9898-cba571ddd664",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wag002/Documents/Wagner_Lab/Projects/vrs-python/venv/3.10/lib/python3.10/site-packages/python_jsonschema_objects/__init__.py:49: UserWarning: Schema version http://json-schema.org/draft-07/schema not recognized. Some keywords and features may not be supported.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ga4gh.vrs import models, vrs_deref, vrs_enref\n",
+    "from ga4gh.core import ga4gh_identify, ga4gh_serialize, ga4gh_digest, ga4gh_deref\n",
+    "\n",
+    "import json\n",
+    "def ppo(o, indent=2):\n",
+    "    \"\"\"pretty print object as json\"\"\"\n",
+    "    print(json.dumps(o.as_dict(), sort_keys=True, indent=indent))\n",
+    "    \n",
+    "from ga4gh.vrs.dataproxy import SeqRepoRESTDataProxy\n",
+    "from biocommons.seqrepo import SeqRepo\n",
```

Comment:
Unused imports + why create function ppo when as_dict() does this for us? 
```suggestion
    "from ga4gh.vrs import models\n",
    "from ga4gh.core import ga4gh_identify\n",
    "\n",
    "import json\n",
    "from ga4gh.vrs.dataproxy import SeqRepoRESTDataProxy\n",
```

Reply:
I think as_dict helps, but ppo also adds some nice indent and sorting for display in the notebook.